packages/react/src/components/OverflowMenuItem/next/OverflowMenuItem.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { match, keys } from '../../../internal/keyboard';
+import { warning } from '../../../internal/warning';
+import * as FeatureFlags from '@carbon/feature-flags';
+import { usePrefix } from '../../../internal/usePrefix';
+
+const OverflowMenuItem = React.forwardRef(function OverflowMenuItem(
+  {
+    className,
+    closeMenu,
+    disabled = false,
+    handleOverflowMenuItemFocus,
+    hasDivider = false,
+    href,
+    isDelete = false,
+    index,
+    itemText = FeatureFlags.enabled('enable-v11-release')
+      ? null
+      : 'Provide itemText',
```

Comment:
I think we could get rid of this conditional here, since `/next` stuff is always for v11